src/styles/blocks/container.scss

```diff
@@ -0,0 +1,38 @@
+@import "./mixin.scss";
+@import "./variables.scss";
+
+.container {
+  display: flex;
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(- 50%, - 50%);
+  justify-content: center;
+  width: 600px;
+  height: 600px;
+  margin: auto;
```

Comment:
`margin `in bem block? no way!) also its looks useless since you use `position`)